raise Exception("hello world")
print("Hello World!")